sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -206,7 +206,7 @@ abstract class QueryTest extends PlanTest {
     val jsonString = try {
       logicalPlan.toJSON
     } catch {
-      case e =>
+      case e: Throwable =>
```

Comment:
Agreed; this change doesn't alter the behavior at all, but it probably should. I'll do that.